### ISSUE TITLE
Let's add more terms to the Glossary

### ISSUE THREAD

**Issue by l0gicgates** [l0gicgates/churning#1]
Please feel free to let me know if I missed any terms or if some do not apply anymore. Happy churning!

**Comment by sanuzi** [comment]
are you still looking for people to update this thing? im down

**Comment by l0gicgates** [comment]
Absolutely. We will need to make some changes to the extension after Reddit made the layout changes
